### PR TITLE
[netcore][10.prompt-validations] Fix typos and minor formatting

### PR DESCRIPTION
**Pull request by VictorGrycuk**
## Proposed Changes
Fix several typos in sample # 10 and minor text format fixes

## Testing
Code behavior was not modified.

### CODE REVIEW

File: samples/csharp_dotnetcore/10.prompt-validations/readme.md [hunk 1]
```diff
@@ -8,17 +8,18 @@ git clone https://github.com/Microsoft/botbuilder-samples.git
 # Prerequisites
 ## Visual Studio
 - Navigate to the samples folder (`botbuilder-samples/samples/csharp_dotnetcore/10.prompt-validations`) and open PromptValidationsBot.csproj in Visual Studio.
```
Comment (joseques) [inline samples/csharp_dotnetcore/10.prompt-validations/readme.md]: Please add `code style` to filenames

**Review by joseques — APPROVED** [review]
LGTM :shipit:

**Review by joseques — APPROVED** [review]
🚢 🇮🇹 
I got a bit confused because of the rebase. :shipit: anyways

**Comment by dfavretto** [comment]
Closed due to Microsoft#769

**Pull request closed without merging**